documentation/whatsnew/fusetools/fusetools-news-11.0.0.AM2.adoc

```diff
@@ -0,0 +1,48 @@
+= Fuse Tooling - What's New in 11.0
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 11.0.0.AM2
+:page-product_id: jbt_core
+:page-product_version: 4.6.0.AM2
+
+== Camel Rest DSL from WSDL wizard
+
+There is a new _"Camel Rest DSL from WSDL"_ wizard. This wizard wraps the wsdl2rest tool now included with the Fuse 7 distribution, which 
+takes a WSDL file for a SOAP-based (JAX-WS) web service and generates a combination of CXF-generated code and a Camel REST DSL route to make it accessible using REST operations. 
+
+To start, you need is an existing Fuse Integration project in your workspace and access to the WSDL for the SOAP service. Then use 
```

Comment:
"you need is" -> "you need" or "your need is" ?

Reply:
1) as per apupier's comment: "you need is" > "you need"
2) All instances of Rest should be "REST" in the UI of the wizard, and then changed here in this What's New document. If the UI of the wizard does not change, then you should leave references to UI text as is in the document.
3) In the UI of the wizard, all instances of "Wsdl" should be "WSDL"
4) Change: " It automatically knows whether your project is Blueprint, Spring, or Spring Boot based, so creates the right artifacts without any additional input."
to:
The wizard determines whether your project is Blueprint, Spring, or Spring Boot based, and it creates the corresponding artifacts without requiring any additional input. 
5) change "and" > "to": "When the wizard is finished, you can open your new Camel file in the Fuse Tooling Route Editor **and** view what it created." to:
When the wizard is finished, you can open your new Camel file in the Fuse Tooling Route Editor **to** view what it created.

Reply:
Change: (if you want tone to be less chatty - nbd)
"You’ll notice we have a new tab in the Fuse Tooling Route Editor called "REST". For now, this tab is read only, but displays a few bits of information:"
To:
The Fuse Tooling Route Editor provides a new **REST** tab. For this release, the contents of this tab is read-only and includes the following information:

Reply:
Change:
Details for the REST Configuration element, which details the component (jetty, netty, servlet, etc), the context path, the port, binding mode (json, xml, etc.), and host. There is only one REST Configuration element.
To
Details for the REST Configuration element including the component (jetty, netty, servlet, etc.), the context path, the port, binding mode (JSON, XML, etc.), and host. There is only one REST Configuration element.

Reply:
Change: A list of REST elements, which collect REST operations. There may be more than one REST element in a given configuration. Each REST element has an associated property page that displays additional details such as the path, data it consumes or produces, etc.

to:
A list of REST elements that collect REST operations. A configuration can have more than one REST element. Each REST element has an associated property page that displays additional details such as the path and the data it consumes or produces.

Reply:
It would be better if screen captures were  limited to the Route Editor view rather than the entire perspective

Reply:
Change:
And a list of REST operations for a given REST element. Each of these has an associated property page that details the URI, output type, and so on.
To:
A list of REST operations for the selected REST element. Each of the operations has an associated property page that provides details such as the URI and output type.

Reply:
Change:
Though this page of the editor and its associated property pages are currently read only, we hope they are useful during application development and we plan on enabling them in a future release. For now, editing REST DSL can occur directly on the Source tab, and when you return to the REST tab it refreshes to show updated details.
To:
For this release, the REST tab is read-only. If you want to edit the REST DSL, use the Route Editor Source tab. When you make changes and save them in the Source tab, the REST tab refreshes to show your updates.